vllm/model_executor/layers/attention/backends/xformers.py

```diff
@@ -109,15 +111,15 @@ def forward(
                 if input_metadata.attn_bias is None:
                     if self.alibi_slopes is None:
                         attn_bias = BlockDiagonalCausalMask.from_seqlens(
-                            [seq_len] * batch_size)
+                            input_metadata.prompt_lens.tolist())
```

Comment:
I think we should avoid converting `prompt_lens` tensor to CPU list inside the model. Can we do it during `prepare_prompts`?